plugins/ui/src/deephaven/ui/_internal/utils.py

```diff
@@ -107,3 +110,60 @@ def remove_empty_keys(dict: dict[str, Any]) -> dict[str, Any]:
         The dict with keys removed.
     """
     return {k: v for k, v in dict.items() if v is not None}
+
+
+def wrapped_callable(
+    max_args: int | str, kwargs_set: set[str] | str, func, *args, **kwargs
+) -> None:
+    """
+    Function that will be called by the dispatcher.
+
+    Args:
+        max_args: The maximum number of positional arguments to pass to the function
+        kwargs_set: The set of keyword arguments to pass to the function
+        func: The function to call
+        *args: args, used by the dispatcher
+        **kwargs: kwargs, used by the dispatcher
+    """
+    args = args if max_args == "any" else args[:max_args]
+    kwargs = (
+        kwargs
+        if kwargs_set == "any"
+        else {k: v for k, v in kwargs.items() if k in kwargs_set}
+    )
+    func(*args, **kwargs)
+
+
+def wrap_callable(func: Callable) -> Callable:
+    """
+    Wrap the function so args are dropped if they are not in the signature.
+
+    Args:
+        func: The callable to wrap
+
+    Returns:
+        The wrapped callable
+    """
+    if sys.version_info.major == 3 and sys.version_info.minor >= 10:
+        sig = signature(func, eval_str=True)
+    else:
+        sig = signature(func)
+
+    max_args = 0
+    kwargs = set()
```

Comment:
I'd prefer the name `kwargs_set` so it's clearer this is not just `kwargs`
```suggestion
    kwargs_set = set()
```